app/views/user_mailer/_due_today.html.erb

```diff
@@ -1 +1 @@
-<p><%= simple_format(replace_variables(@app_configs.upcoming_checkin_email_body)) %></p>
\ No newline at end of file
+<%= render html: simple_format(replace_variables(@app_configs.upcoming_checkin_email_body)).gsub('\n', '').html_safe %>

```

Comment:
Not necessary, just replace the `\n`'s with actual line breaks... simple :smile:. We still need `simple_format` since we need to allow admins to enter their own copy, but we can definitely skip the `gsub`.